CRM/Extendedreport/Form/Report/ExtendedReport.php

```diff
@@ -6322,6 +6322,21 @@ protected function alterCampaignType($value): string {
    *   Event label.
    */
   protected function alterEventType($value): string {
+    // Check if $value is comma separated.
+    $separator = ',';
+    if (strpos($value, $separator) !== FALSE) {
```

Comment:
non blocking - but you could have just exploded without the condition - then if there is no comma you would still wind up with an array of one value